programs/registry/src/account_compression_cpi/register_program.rs

```diff
@@ -25,7 +25,7 @@ pub struct RegisterProgram<'info> {
     pub registered_program_pda: AccountInfo<'info>,
     /// CHECK: (account compression program). TODO: check that a signer is the upgrade authority.
```

Comment:
remove todo?

Reply:
want to keep this todo for a next release